### PR TITLE
Add webpack + regenerate packages with npm2nix.

### PR DESCRIPTION
**Pull request by teh**
npm2nix ./pkgs/top-level/node-packages.json  ./pkgs/top-level/node-packages-generated.nix

### CODE REVIEW

File: pkgs/top-level/node-packages.nix [hunk 1]
```diff
@@ -25,6 +25,15 @@ rec {
     "node-protobuf".buildInputs = [ pkgs.protobuf ];
     "rbytes".buildInputs = [ pkgs.openssl ];
 
+    # Remove fsevents dependency because it is OSX-specific.
```
Comment (offlinehacker) [inline pkgs/top-level/node-packages.nix]: Ohh, that's prone to breaking, i should implement support for generating for platforms. But isn't fsevents part of optionalDependencies, and currently npm2nix does not even handle optionalDependencies.

Comment (teh) [inline pkgs/top-level/node-packages.nix]: Hm yea, it's optional [1] but it shows up as `"fsevents-0.3.5" = self.by-version."fsevents"."0.3.5";` in the npm2nix output.. I can retract this PR and wait for platform support? webpack is not urgent at all, I can just keep a local version around.

[1]
https://github.com/paulmillr/chokidar/blob/master/package.json

**Comment by offlinehacker** [comment]
Fixed in https://github.com/NixOS/nixpkgs/commit/d84a4f5699f774e75f427541fc0e21cac41e2d15

**Comment by offlinehacker** [comment]
Package was added of course and it works.

**Comment by teh** [comment]
Great stuff, thanks!

**Pull request closed without merging**